sunpy/coordinates/ephemeris.py

```diff
@@ -61,7 +61,9 @@ def get_body_heliographic_stonyhurst(body, time='now', observer=None, *, include
     -----
     There is no correction for aberration due to observer motion.  For a body close to the Sun in
     angular direction relative to the observer, the correction can be negligible because the
-    apparent location of the body will shift in tandem with the Sun.
+    apparent location of the body will shift in tandem with the Sun. As Astropy's built-in ephemeris
+    is not as accurate as JPL ephemeris so you can use `~astropy.coordinates.solar_system_ephemeris`
```

Comment:
I assume we would want to link to https://docs.astropy.org/en/stable/coordinates/solarsystem.html as well.

It has the ref, `astropy-coordinates-solarsystem` we should be able to link that directly.

Reply:
`~astropy.coordinates.solar_system_ephemeris`
I guess clicking on this link will open https://docs.astropy.org/en/stable/coordinates/solarsystem.html .
Doesn't it?

Reply:
Did you build the documentation and check? 

I don't think that will link that page but the API page for `solar_system_ephemeris`

Reply:
I got it